clusters/app.ci/cert-manager/cert-manager_ns.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cert-manager
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cert-manager-system
```

Comment:
Why not put the leader election resources in the cert-manager namespace as well? Then we have one namespace for everything

Reply:
I do not know why `kube-system` is used in the first place.
Let me try to put everything into one place.

Reply:
Seems working.